dv/uvm/core_ibex/scripts/compile-tb.py

```diff
@@ -35,6 +37,32 @@ def main() -> int:
 
     os.makedirs(args.output, exist_ok=True)
 
+    # Setup environment variables for the tb to link against the compiled ISS
+    if args.en_cosim:
+        _spike_iss_pc = 'riscv-riscv riscv-disasm riscv-fdt'
+
+        if not subprocess.run(shlex.split(f'pkg-config --exists {_spike_iss_pc}'),
```

Comment:
Probably easier to make `_spike_iss_pc` a list of strings and then call something like `subprocess.run(['pkg-config', '--exists'] + _spike_iss_pc, ...)`?